packages/components/src/components/icon/icon.tsx

```diff
@@ -89,6 +89,14 @@ export class Icon {
     );
 
     // TODO: wouldn't it be better to set alt attribute instead of aria-label?
-    return <img src={buildIconUrl(this.source || this.name)} {...parseAndGetAriaAttributes(this.aria)} alt="" />;
+    return (
+      <img
+        src={buildIconUrl(this.source || this.name)}
+        {...parseAndGetAriaAttributes(this.aria)}
+        {...(!this.source && { width: 24, height: 24 })} // improve bootstrapping behaviour
```

Comment:
Why conditional?

Reply:
We should maybe not add the width and height attributes for custom icons 🤔

Reply:
As discussed in private chat, we should add the attributes also for custom sources